SU-SWS/stanford_basic
# D8CORE-2454: Lockup default array structure mis-match fix.

# READY FOR REVIEW 

# Summary
- Set the default lockup option to option A buy setting the correct array structure. 

# Review By (Date)
- EOD SEPT 9

# Urgency
- 11/10

# Steps to Test

1. Do a fresh install of Cardinal sites
2. See lockup a little off center
3. Inspect markup and see missing `option-a` class
4. Check out this branch and clear cache
5. See default option a.

# Affected Projects or Products
- D8 CORE
- SOE
- SA/OOD

# Associated Issues and/or People
- D8CORE-2454

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
